rust/tools/authorization-logic/BUILD

```diff
@@ -14,10 +14,10 @@
 # limitations under the License.
 #
 #-------------------------------------------------------------------------------
-package(default_visibility = ["//visibility:public"])
-
 load("@rules_rust//rust:rust.bzl", "rust_binary", "rust_library", "rust_test")
 
+package(default_visibility = ["//visibility:public"])
```

Comment:
This is a change based on linter feedback

Reply:
I see. I'm not sure that this is a good idea: when we import these things into the Google internal repo, the checks will get mad at us for having `public` visibility. If you want something visible throughout all of Raksha, try doing `default_visibility = ["//:__subpackages__"]`, which is basically the same out here but won't make the internal checks upset.

Reply:
Thank you! That's what I was hoping the 'public' setting would do :)